src/main/java/frc/robot/RobotContainer.java

```diff
@@ -181,6 +182,24 @@ private void configureButtonBindings() {
         }));
     }
 
+    public BaseSwerveSubsystem getSwerveSubsystem(){
+        if (driveSubsystem instanceof BaseSwerveSubsystem){
+            return (BaseSwerveSubsystem) driveSubsystem;
+        }
+
+        else{
+            return null;
+        }
+    }
+
+    public GripperSubsytem getGripperSubsytem(){
```

Comment:
You probably don't need these getters. It's more explicit to pass in the subsystems directly.